src/modules/reporting/components/reporting-report-confirm/reporting-report-confirm.jsx

```diff
@@ -42,6 +47,9 @@ ReportingReportConfirm.propTypes = {
   selectedOutcome: PropTypes.string.isRequired,
   stake: PropTypes.string.isRequired,
   isMarketValid: PropTypes.bool,
+  designatedReportNoShowReputationBond: PropTypes.string,
```

Comment:
Is this the correct type?

Same for `reporterGasCost`.

Reply:
It's the same as reporter gas cost, It's actually a formatted number object. I'll make the change to object.